tests/posix/common/src/stropts.c

```diff
@@ -18,6 +18,19 @@ ZTEST(stropts, test_putmsg)
 	zassert_equal(errno, ENOSYS, "Expected errno ENOSYS, got %d", errno);
 }
 
+ZTEST(stropts, test_putpmsg)
+{
+	const struct strbuf *ctrl = NULL;
+	const struct strbuf *data = NULL;
+	int fd = -1;
+	int band = 0;
+	int flags = 0;
+	int ret = putpmsg(fd, ctrl, data, band, flags);
+
+	zassert_equal(ret, -1, "Expected return value -1, got %d", ret);
+	zassert_equal(errno, ENOSYS, "Expected errno ENOSYS, got %d", errno);
+}
```

Comment:
Same here - it might be better to keep it short and sweet.
```suggestion
ZTEST(stropts, test_putpmsg)
{
	zassert_equal(putpmsg(-1, NULL, NULL, 0, 0), -1, "Expected return value -1, got %d", ret);
	zassert_equal(errno, ENOSYS, "Expected errno ENOSYS, got %d", errno);
}
```

Reply:
Should I update the others stubs ?

Reply:
No, just the ones involved in this PR